homeassistant/components/esphome/sensor.py

```diff
@@ -1,4 +1,6 @@
 """Support for esphome sensors."""
+from __future__ import annotations
```

Comment:
We import the names from the library at the top so looks like this isn't needed.